src/main/java/reposense/system/DashboardServer.java

```diff
@@ -0,0 +1,56 @@
+package reposense.system;
+
+import java.awt.Desktop;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import net.freeutils.httpserver.HTTPServer;
+
+/**
+ * Handles starting of the server to display the dashboard.
+ */
+public class DashboardServer {
+
+    private static final String LOCAL_HOST_URL = "http://localhost:%s/";
+
+    private static final Logger logger = LogsManager.getLogger(DashboardServer.class);
+
+    /**
+     * Starts a server at {@code port} and loads the {@code requestPath} from the local disk.
+     */
+    public static void startServer(int port, Path requestPath) {
+        logger.info("Starting a server at port " + port + "...");
+        HTTPServer server = new HTTPServer(port);
+
+        // default virtual host
+        HTTPServer.VirtualHost host = server.getVirtualHost(null);
+
+        try {
+            // a handler to process the request and give the corresponding response
+            host.addContext("/", new HTTPServer.FileContextHandler(requestPath.toFile()));
+            server.start();
+            launchBrowser(String.format(LOCAL_HOST_URL, port));
+        } catch (IOException ioe) {
+            logger.log(Level.SEVERE, ioe.getMessage(), ioe);
+        }
+    }
+
+    /**
+     * Launches the default browser with {@code url}.
+     */
+    private static void launchBrowser(String url) throws IOException {
+        try {
+            if (Desktop.isDesktopSupported()) {
+                Desktop.getDesktop().browse(new URI(url));
+                logger.info("Loading " + url + " on the default browser...");
+                logger.info("Press Ctrl + C or equivalent to exit");
+            }
```

Comment:
Add a `else` here, and log (severe) that we can't this class/library is not supported for the user's pc, and ask the user to manually refer to user guide for other ways to view the dashboard.
Should let the user know if they somehow can't start the server through Reposense, rather then leave them hanging there wondering what happened.